content/reference-data-api.md

```diff
@@ -3006,6 +3212,190 @@ Response:
 
 
 
+## Get Account Transaction Stats ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/accountStats.js "Source")
+
+Retrieve daily summaries of transaction activity for an account. _(New in [v2.1.0][].)_
+
+<!--<div class='multicode'>-->
+
+*REST*
+
+```
+GET /v2/accounts/{:address}/stats/transactions
+```
+
+<!--</div>-->
+
+[Try it! >](data-api-v2-tool.html#get-account-transaction-stats)
+
+This method requires the following URL parameters:
+
+| Field    | Value  | Description |
+|----------|--------|-------------|
+| :address | String | Ripple address to query |
+
+
+Optionally, you can also include the following query parameters:
+
+| Field      | Value   | Description |
+|------------|---------|-------------|
+| start      | String  | UTC start time of query range. Defaults to start of current date. |
+| end        | String  | UTC end time of query range. Defaults to current date. |
+| limit      | Integer | Max results per page (defaults to 200). Cannot be more than 1000. |
+| descending | Boolean | If true, sort results with most recent first. By default, sort results with oldest first. |
```

Comment:
I _think_ that should be accurate, but I'm hesitant to make this change across many API calls because there may be some weird cases where the API checks for the presence of the parameter rather than its value. (I don't _think_ the Data API has any of that, but there's at least one case in the `rippled` APIs, so I'm wary...)

I'm going to leave this as is for consistency with the rest of the document, rather than investigating every API method just to be sure.